src/VisualStudio/Core/Test/EditAndContinue/VsReadOnlyDocumentTrackerTests.vb

```diff
@@ -16,13 +16,14 @@ Imports Microsoft.VisualStudio.Utilities
 Imports Roslyn.Test.Utilities
 
 Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.EditAndContinue
+#If False Then
```

Comment:
❔ Why `#If False` instead of `WpfFact(Skip:="bug link")`?

Reply:
Deleted the file.